site/docs/plugins/emoji.md

```diff
@@ -1 +1,68 @@
-Coming soon, please come back later
+# Emoji Plugin (`emoji`)
+
+With this plugin, you can easily insert emojis on your replies searching for them and inserting text instead of manually copying an emoji from web or Telegram and pasting at your code.
```

Comment:
I'd extend this part. It should first say that
- people use emoji in their code
- why that's bad/annoying (hard to edit on desktop, need to copy & paste from internet, not all desktop systems can render all emojis)
- how this plugin solves it (externalise emoji data, reference by name)
- uses template strings to have a super concise syntax (link to MDN, it's not a well-known features in JS)
- autocomplete is available!

Some of these points are already touched in the current version, but they're mixed in one long and hard to read sentence.

You may want to decide to only elaborate on a few of these points, then show the code examples early, and then explain the rest.